Calendar/Get-RBASummary.ps1

```diff
@@ -92,14 +92,14 @@ function ValidateInboxRules {
         exit
     } elseif ($rules.Name -like "REDACTED-*") {
         Write-Host -ForegroundColor Yellow "Warning: No PII Access to MB so cannot check for Delegate Rules."
-        Write-Host -ForegroundColor Red " --- Inbox Rules needs to be checked manually for any Delegate Rules. --"
         Write-Host -ForegroundColor Yellow "To gain PII access, Mailbox is located on $($mailbox.Database) on server $($mailbox.ServerName)"
-        if ($rules.count -eq 1) {
+        if ($null -eq $rules.count) {
```

Comment:
I think it would be better to include both statements here. 

```
$null -eq $rules.count -or $rules.count -eq 1
``

This way if `$rules.count` does equal 1 we catch it here.

Reply:
PS is weird if you have an "Array" with count of 1.  $rules.count will not return any value, if there is only 1 value. 
Let me look if I can predefine it as an array to by pass this weirdness.

Reply:
Yeah, it is auto defining the then as NOT array, so count doesn't exist. So, the best way to handle this is by making it always be an array. 

```
$rules = @(Get-InboxRule)

OR 

[array]$rules = Get-InboxRule
```

Reply:
Ah... now I see what you are saying. 
-Fixed.

Reply:
Belt and suspenders now.